platform/default/glfw_view.cpp

```diff
@@ -164,9 +165,12 @@ void GLFWView::onKey(GLFWwindow *window, int key, int /*scancode*/, int action,
         case GLFW_KEY_Q:
             view->clearAnnotations();
             break;
-        case GLFW_KEY_P: {
+        case GLFW_KEY_P:
             view->addRandomCustomPointAnnotations(1);
-        } break;
+            break;
+        case GLFW_KEY_L:
+            view->addRandomLineAnnotations(1);
```

Comment:
shouldn't this be called `addRandomCustomLineAnnotations` for consistency with Point?